fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -1729,7 +1730,8 @@ def _wait_for_post_backward(self) -> None:
         if self._require_backward_grad_sync:
             # Flush any unreduced buckets in the post_backward stream.
             with torch.cuda.stream(self._streams["post_backward"]):
-                assert self._reducer is not None
+                p_assert(self._reducer is not None, "WFPB: reducer is None")
```

Comment:
What does WFPB mean?

Reply:
it is Wait-For-Post-Backward hahaha ;-)

Reply:
Hahah I couldn't figure it out either. maybe we should spell it out for future readers.